compiler/rustc_mir/src/interpret/traits.rs

```diff
@@ -99,8 +105,8 @@ impl<'mir, 'tcx: 'mir, M: Machine<'mir, 'tcx>> InterpCx<'mir, 'tcx, M> {
     }
 
     /// Resolves the function at the specified slot in the provided
-    /// vtable. An index of '0' corresponds to the first method
-    /// declared in the trait of the provided vtable.
+    /// vtable. Currently an index of '3' (`COMMON_VTABLE_ENTRIES.len()`)
+    /// corresponds to the first method declared in the trait of the provided vtable.
```

Comment:
This function still does `idx.checked_add(3)`; that is probably wrong then.

Reply:
Oops, indeed!